packages/datadog-plugin-cypress/src/plugin.js

```diff
@@ -212,6 +218,16 @@ module.exports = (on, config) => {
       testSpanMetadata[TEST_CODE_OWNERS] = codeOwners
     }
 
+    if (isUnskippable) {
+      hasForcedToRunTests = true
```

Comment:
Isn't this true only if the test was going to be skipped?

Reply:
Or... should this be `hasUnskippableTests = True` ? 🧐

Reply:
great catch! I don't know how the integration test didn't catch this. I'll check, thanks!

Reply:
ok I know how. I'm missing session and module checks. I think I left them out until I got the other PRs merged but once they were merged I forgot to add them here 😅

Reply:
fixed and added a test in https://github.com/DataDog/dd-trace-js/pull/3684/commits/40ce56fe99741d7538102b630d247cf95f6ba293